docs/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md

```diff
@@ -9,7 +9,7 @@ import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
 [Deploying Camunda 8 using Helm charts](../deploy.md) will work for all platforms and distributions. However, each platform or cloud provider may have special prerequisites or pitfalls to avoid. This section highlights important notes for a smooth Camunda 8 deployment on different Kubernetes platforms.
 
-In addition to Stock Kubernetes (which could be deployed on cloud or on-premise), Camunda only officially tests and supports the following platforms:
+In addition to [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/) (which could be deployed on cloud or on-premise), Camunda only officially tests the following platforms:
```

Comment:
Stock Kubernetes (or better let's call it `upstream` Kubernetes) is the parent of all CNCF Certified Kubernetes distributions. So we need to mention both.